src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
+    private static void logoWithHello() {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____\n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+
+    public static void horizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void greet() {
+        horizontalLine();
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?\n");
+        horizontalLine();
+    }
+
+    public static void exit() {
+        horizontalLine();
+        System.out.println("Bye. Hope to see you again soon!\n");
+        horizontalLine();
+    }
+
+    public static void echo(String userInput) {
+        horizontalLine();
+        System.out.println("added: " + userInput);
+        horizontalLine();
+    }
+
+    private static void listCommand(Task[] tasks) {
+        horizontalLine();
+        for (int i = 0; i < Task.getNumberOfTasks(); i += 1) {
+            System.out.println(tasks[i].getTaskNumber() + "." + tasks[i]);
+        }
+        horizontalLine();
+    }
+
+    public static String[] processLine(String userInput) {
+        String[] words = userInput.split(" ");
+        return words;
+    }
+
+    public static void taskAdded(Task t) {
+        horizontalLine();
+        boolean lessThanOne = (Task.getNumberOfTasks() <= 1);
```

Comment:
- [ ] Why will 1 be included in this boolean variable? Supposed that `Task.getNumberOfTasks() == 1`, the value would be true and **not less than one**. Perhaps you would like to review this portion of the code to see if this is the intended functionality as it might cause bugs, or change the naming as it might cause confusion in the future.
- [ ] Would you like to consider removing the brackets as it is unnecessary?
- [ ] Would you like to change the variable name to make it sound boolean instead?
```suggestion
        boolean isLessThanOne = Task.getNumberOfTasks() <= 1;
```